src/main/resources/uischema.json

```diff
@@ -78,7 +86,7 @@
             "value": {
               "name": "sequential",
               "displayName": "Execute sequentially",
-              "inputType": "boolean",
```

Comment:
There are other instances where boolean is used in this file for the input type. Shall we update them as well?

Reply:
Fixed with e38d9d1